README.md

```diff
@@ -70,26 +70,113 @@ or you can directly download [sources](https://github.com/Zenika/immutadot/relea
 
 ## Usage
 
-in browser:
+ES modules:
 
 ```js
-  import { push } from 'immutadot'
+import { set } from 'immutadot'
+```
+
+CommonJS:  
 
-  push({ nested: { prop: [1, 2] } }, 'nested.prop', 3, 4)
-  // → { nested: { prop: [1, 2, 3, 4] } }
+```js
+const { set } = require('immutadot')
 ```
 
-in node:
+### Example
+
+Object used in the following example:
 
 ```js
-  const { push } = require('immutadot')
+const animals = {
+  weasel: [
+    {
+      vernacularName: 'badger',
+      scientificName: 'Meles meles'
+    },
+    {
+      vernacularName: 'otter',
+      scientificName: ''
+    }
+  ]
+}
+```
 
-  push({ nested: { prop: [1, 2] } }, 'nested.prop', 3, 4)
-  // → { nested: { prop: [1, 2, 3, 4] } }
+We forgot to write down otter's scientific name. Let's set this property with the right name ```Lutrinae```.
```

Comment:
What about : "Let's add the otter's scientific name without mutating the original object structure :"